AWS/aws.js

```diff
@@ -117,16 +123,20 @@ const NEPTUNE_DB_CLUSTER_PARAMETER_GROUP =
 const ECS_TASK_DEFINITION = "AWS::ECS::TaskDefinition";
 const WAF_RULE_GROUP = "AWS::WAFv2::RuleGroup";
 const EKS_NODE_GROUP = "AWS::EKS::NodeGroup";
+const COMPUTE_OPTIMIZER_ENROLLMENT_STATUS =
```

Comment:
dont forget to add the new resource type here too

Reply:
AWS::AutoScaling::ScalingPolicy is added directly to aws-dependencies.json actually